codecov.yml

```diff
@@ -15,16 +8,14 @@ coverage:
       frontend:
         # codecov will not fail status checks for master
         only_pulls: true
-        # This will be needed until codecov supports after_n_builds for flags
-        informational: true # Do not fail the check
+        informational: false # Fail the check
```

Comment:
The codecov CI checks will be back and will fail if they do not hit a 60% FE coverage.